study.md

```diff
@@ -43,5 +51,12 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+
+A Hash URL is like `http://thingy.com/#/posts/1/history`. They have `#` in the URL.
```

Comment:
Yes, that is an example of a Hash URL.  Hash URLs aren't good for copying and pasting, because the server drops the hash and looses the view that was being shared.